src/scopes/grants/recipientsWithoutTTA.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/prefer-default-export */
+import { Op } from 'sequelize';
+import { sequelize } from '../../models';
+
+const grantsMissingActivitySql = (beginActivityDate, finishActivityDate) => sequelize.literal(
+  `
+    -- Determine what grants have activity reports within the given date range.
```

Comment:
can we make these JS comments? That way they don't get printed in the server logs everytime someone uses the filters